web/src/components/User/UserForm/UserForm.tsx

```diff
@@ -231,7 +230,7 @@ const UserForm = (props: UserFormProps) => {
                     id="userIsActive"
                     className="form-check-input"
                     defaultChecked={value}
-                    disabled={disabled}
+                    disabled={!user}
```

Comment:
curious- what is the purpose of this disabled? Are there specific user-types that can access User-edit but _not_ activate/de-activate users? If not, would it be better to just remove the `disabled=...` attribute entirely?

Reply:
this is only to disable the user status field (Active/Inactive) when creating a new user, forcing a status of Active. The status field is enabled for any user able to edit a user